### ISSUE TITLE
Watchify doesn't work

### ISSUE THREAD

**Issue by aldendaniels** [aldendaniels/browserify-split-vendor-modules#6]
On change I get the following:

```
fs.js:491
  binding.open(pathModule._makeLong(path),
          ^
TypeError: path must be a string
    at TypeError (native)
    at Object.fs.open (fs.js:491:11)
    at OpenReq.process (/Users/alden/Source/browserify-split-vendor-modules/node_modules/watchify/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/graceful-fs.js:55:16)
    at OpenReq.Req (/Users/alden/Source/browserify-split-vendor-modules/node_modules/watchify/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/graceful-fs.js:126:8)
    at new OpenReq (/Users/alden/Source/browserify-split-vendor-modules/node_modules/watchify/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/graceful-fs.js:49:7)
    at Object.open (/Users/alden/Source/browserify-split-vendor-modules/node_modules/watchify/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/graceful-fs.js:41:3)
    at ReadStream.open (fs.js:1613:6)
    at new ReadStream (fs.js:1600:10)
    at Object.fs.createReadStream (fs.js:1554:10)
    at Deps.readFile (/Users/alden/Source/browserify-split-vendor-modules/node_modules/browserify/node_modules/module-deps/index.js:190:17)
```

**Comment by aldendaniels** [comment]
@jmm - Do you think this is likely to be a PathModify issue?

**Comment by aldendaniels** [comment]
Fixed in https://github.com/aldendaniels/browserify-split-vendor-modules/commit/ef31f769bb8166aac3372026e9352bd1f5eb244a.

**Comment by jmm** [comment]
> Fixed in ef31f76.

Ah, ok. Yeah, there's a hokey relationship between `id` and `file` right now that causes a lot of inconsistencies. I'd like to fix all of that eventually.
